container-core/src/test/java/com/yahoo/container/handler/metrics/PrometheusHandlerTest.java

```diff
@@ -0,0 +1,131 @@
+// Copyright 2019 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.container.handler.metrics;
+
+import com.github.tomakehurst.wiremock.junit.WireMockRule;
+import com.yahoo.container.jdisc.RequestHandlerTestDriver;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+
+import java.io.BufferedReader;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.concurrent.Executors;
+import java.util.stream.Collectors;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.equalTo;
+import static com.github.tomakehurst.wiremock.client.WireMock.get;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlPathEqualTo;
+import static com.github.tomakehurst.wiremock.core.WireMockConfiguration.options;
+import static com.yahoo.container.handler.metrics.MetricsV2Handler.V2_PATH;
+import static com.yahoo.container.handler.metrics.MetricsV2Handler.VALUES_PATH;
+import static com.yahoo.container.handler.metrics.MetricsV2Handler.consumerQuery;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+/**
+ * @author gjoranv
+ */
+public class PrometheusHandlerTest {
+
+    private static final String URI_BASE = "http://localhost";
+
+    private static final String V2_URI = URI_BASE + PrometheusHandler.V1_PATH;
+    private static final String VALUES_URI = URI_BASE + PrometheusHandler.VALUES_PATH;
+
+    // Mock applicationmetrics api
+    private static final String MOCK_METRICS_PATH = "/node0";
+
+    private static final String TEST_FILE = "application-metrics.json";
```

Comment:
I suggest to use a prometheus specific test file, even though this handler should just directly output the data it gets from the metrics-proxy. (The output from the ApplicationMetricsHandler in the real world should be the metrics for all nodes in prometheus format.)